modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2014,23 +2014,51 @@ test.describe('Manage object entries through Workflow', () => {
 });
 
 scheduleTest.describe('Manage object entries schedule properties', () => {
-	scheduleTest(
-		'can create, read, update, and delete a reviewDate of an object entry',
-		async ({apiHelpers, page, viewObjectEntriesPage}) => {
-			const objectDefinition =
-				await apiHelpers.objectAdmin.postRandomObjectDefinition({
-					status: {code: 0},
-				});
+	let _objectDefinition: ObjectDefinition;
```

Comment:
Hi @larissacribeiro, this private markup is commonly seen in the backend. Why bring these patterns into a frontend functional test?

Reply:
Hey Igor, this pattern is already in this file so I believed it would be more interesting to follow it. Or, maybe we can change the existing one, what do you think?

Reply:
I understand, thanks for the explanation. We don't need to change this pattern for the whole file now. We can continue like this